README.md

```diff
@@ -89,21 +184,20 @@ Here are a set of tutorials that will help you get started:
 | `Azure IoT Device Workbench: Compile Device Code`  | Compile device code. |
 | `Azure IoT Device Workbench: Upload Device Code`  | Compile and upload device code. |
 | `Azure IoT Device Workbench: Configure Device Settings...`  | Manage the settings on the device. |
-| `Azure IoT Device Workbench: Install Device Compiler Toolchain...`  | Install device compiler toolchain. |
 | `Azure IoT Device Workbench: Set Workbench Path` | Set the default path for Azure IoT Device Workbench. |
 | `Azure IoT Device Workbench: Help` | Get help for Azure IoT Device Workbench. |
 
 ### Create Project
 
-<img width="760" src="https://raw.githubusercontent.com/Microsoft/vscode-iot-workbench/master/gif/new.gif">
+![New project](./docs/images/new.gif)
 
 ### Provision Azure Services
 
-<img width="760" src="https://raw.githubusercontent.com/Microsoft/vscode-iot-workbench/master/gif/provision.gif">
+![Provision Azure services](./docs/images/provision.gif)
 
 ### Open Examples
 
-<img width="760" src="https://raw.githubusercontent.com/Microsoft/vscode-iot-workbench/master/gif/example.gif">
+![Open samples](./docs/images/example.gif)
 
 #### Note: 
 -  When invoking the **Azure IoT Device Workbench: Provision Azure Services...** command with Azure IoT Hub and Azure Functions, by default, Azure Functions would use the IoT Hub consumer group of `$Default`. To switch to another consumer group, please follow the [guide](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-create-through-portal) to create a new consumer group in Azure Portal. Then in IoT project, modify the following setting in **function.json**:
```

Comment:
In the `You may open an existing IoT project after you close it. ` section, for container project, user should use *File -> Open Folder...* to open a project.